pydarn/plotting/rtp.py

```diff
@@ -417,7 +435,9 @@ def plot_range_time(cls, dmap_data: List[dict], parameter: str = 'v',
         else:
             tick_interval = 1
         ax.xaxis.set_minor_locator(dates.MinuteLocator(interval=tick_interval))
-        if coord is Coord.SLANT_RANGE:
+        if coord is Coords.SLANT_RANGE:
+            ax.yaxis.set_minor_locator(ticker.AutoMinorLocator(2))
+        if coord is Coords.GROUND_SCATTER_MAPPED_RANGE:
             ax.yaxis.set_minor_locator(ticker.AutoMinorLocator(2))
         else:
             ax.yaxis.set_minor_locator(ticker.MultipleLocator(5))
```

Comment:
sorry for the notification, just amending to follow Marina's suggestion on the other one
```suggestion
        if coord is Coords.SLANT_RANGE or coord is Coords.GROUND_SCATTER_MAPPED_RANGE:
            ax.yaxis.set_minor_locator(ticker.AutoMinorLocator(2))
        else:
            ax.yaxis.set_minor_locator(ticker.MultipleLocator(5))
```

Reply:
And in doing so I deleted the actual comment - this also fixes the bug that was created by me in the first place (sorry, see the if instead of elif on line 440 pre changes - that was causing the bug)